src/EventStore.Client/EventStore.Client.csproj

```diff
@@ -6,10 +6,12 @@
 		<PackageId>EventStore.Client.Grpc</PackageId>
 	</PropertyGroup>
 	<ItemGroup>
-		<PackageReference Include="Google.Protobuf" Version="3.21.9" />
-		<PackageReference Include="Grpc.Net.Client" Version="$(GrpcPackageVersion)" />
-		<PackageReference Include="Microsoft.Extensions.Logging" Version="7.0.0" />
-		<PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" PrivateAssets="All" />
+		<PackageReference Include="Google.Protobuf" Version="3.25.1" />
+		<PackageReference Include="Grpc.Net.Client" Version="2.59.0" />
+<!--        <PackageReference Include="Grpc.Net.Client" Version="$(GrpcPackageVersion)" />-->
+		<PackageReference Include="Microsoft.Extensions.Logging" Version="8.0.0" />
+<!--		<PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" PrivateAssets="All" />-->
+        <PackageReference Include="Microsoft.SourceLink.GitHub" Version="8.0.0" PrivateAssets="All" />
```

Comment:
possibly still in the process of neatening these bits up?

Reply:
(and corresponding part in Directory.Build.Props)

Reply:
some leftovers, on it